doc/test-framework/writing-e2e-tests.md

```diff
@@ -101,19 +101,26 @@ The next step is to create a TestCtx for the current test and defer its cleanup
 
 ```go
 ctx := framework.NewTestCtx(t)
-defer ctx.Cleanup(t)
+defer ctx.Cleanup()
 ```
 
-Now that there is a TestCtx, the test's kubernetes resources (specifically the test namespace,
-RBAC, and Operator deployment) can be initialized:
+Now that there is a `TestCtx`, the test's kubernetes resources (specifically the test namespace,
+Service Account, RBAC, and Operator deployment in `local` testing; just the Operator deployment
+in `cluster` testing) can be initialized:
 
 ```go
-err := ctx.InitializeClusterResources()
+err := ctx.InitializeClusterResources(&framework.CleanupOptions{TestContext: ctx, Timeout: cleanupTimeout, RetryInterval: cleanupRetryInterval})
 if err != nil {
     t.Fatalf("failed to initialize cluster resources: %v", err)
 }
 ```
 
+The `InitializeClusterResources` function uses the custom `Create` function in the framework client to create the resources provided
+in your namespaced manifest. The custom `Create` function use the controller-runtime's client to create resources and then
+creates a cleanup function that is called by `ctx.Cleanup` which deletes the resource and then waits for the resource to be
+fully deleted before returning. This is configurable with `CleanupOptions`. If the `CleanupOptions` argument is set
+to `nil`, the cleanup function simply calls the delete function and returns without waiting.
```

Comment:
> If the `CleanupOptions` argument is set
to `nil`, the cleanup function simply calls the delete function and returns without waiting.

Is this still true. I thought if `CleanupOptions == nil` then there is no cleanup as mentioned below.

Reply:
Add a link to the cleanup options section below so users can see how to configure cleanup options.